client/components/map/mapToolTip.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react'
 import { Popup } from 'react-leaflet'
-const Tooltip = ({distance, trailName, ...props}) => (
+const Tooltip = ({distance, trailName, isActive, ...props}) => (
```

Comment:
I believe that those props do not contain `isActive` if I remember right. I think those are the props passed down from the `Popup`